intermediate_html_css/grid/creating_a_grid.md

```diff
@@ -168,6 +168,7 @@ Now that you’ve made a grid you can start to see how easy it is to control the
 
 <div class="lesson-content__panel" markdown="1">
 - Read Parts I, II and III from [CSS-Tricks Complete Guide to Grid.](https://css-tricks.com/snippets/css/complete-guide-grid/)
+- Watch this [short video](https://www.youtube.com/watch?v=0m5qgfX2TVQ&ab_channel=PeterSommerhoff) on grid terminology from PeterSommerhoff.
```

Comment:
Can this be moved to additional resources please? I think it would be useful for learners who want a different explanation from the articles, but that should be something that is optional for learners.

Reply:
Hi @KevinMulhern,

I appreciate your feedback! I've made the requested changes by moving the grid terminology video   to the "Additional Resources" section. and now  learners now have these as optional, alternative explanations.

Let me know if there's anything else you'd like me to adjust.

Thanks!